src/config/configManager.go

```diff
@@ -182,12 +182,14 @@ func LoadConfigFromFile() {
 	}
 
 	CurrentCfg.ConfigObservability = types.ConfigObservability{
```

Comment:
Do not bring this under observability config, bring this under generic config.

Reply:
Changes are made and created new generic config.